contrib/openssl-cmake/CMakeLists.txt

```diff
@@ -199,6 +202,13 @@ elseif(ARCH_RISCV64)
     perl_generate_asm(${OPENSSL_SOURCE_DIR}/crypto/riscv64cpuid.pl ${OPENSSL_BINARY_DIR}/crypto/riscv64cpuid.S)
     perl_generate_asm(${OPENSSL_SOURCE_DIR}/crypto/aes/asm/aes-riscv64-zkn.pl ${OPENSSL_BINARY_DIR}/crypto/aes/aes-riscv64-zkn.S)
     perl_generate_asm(${OPENSSL_SOURCE_DIR}/crypto/modes/asm/ghash-riscv64.pl ${OPENSSL_BINARY_DIR}/crypto/modes/ghash-riscv64.S)
+elseif(ARCH_LOONGARCH64)
+    macro(perl_generate_asm FILE_IN FILE_OUT)
+        add_custom_command(OUTPUT ${FILE_OUT}
```

Comment:
Please apply the same changes as done [here](https://github.com/ClickHouse/ClickHouse/pull/62754) to the invocation of `add_custom_command` - thanks.

Reply:
Ok.